spec/Creator/RefundUnitsCommandCreatorSpec.php

```diff
@@ -25,21 +25,24 @@
 
 final class RefundUnitsCommandCreatorSpec extends ObjectBehavior
 {
-    public function let(
-        UnitRefundTotalCalculatorInterface $unitRefundTotalCalculator
-    ): void {
-        $this->beConstructedWith($unitRefundTotalCalculator);
+    public function let(RefundUnitsConverterInterface $refundUnitsConverter): void
```

Comment:
`public` in spec 💃

Reply:
Why not? `public` is a default one if there is not access modifier specified